lsp-mode.el

```diff
@@ -4939,15 +4939,21 @@ RENDER-ALL - nil if only the signature should be rendered."
 (declare-function posframe-hide "ext:posframe")
 (declare-function posframe-poshandler-point-bottom-left-corner-upward "ext:posframe")
 
-(defvar lsp-signature-posframe-params
+(defface lsp-signature-posframe
```

Comment:
dropping lsp-signature-posframe-params removes the possibility to customize other properties, e. g. width.

Reply:
Well, we aren't customizing those either now right? When there's a need, someone somewhere will introduce a few new deffaces for the sizes.

Reply:
Users could still change the value of that var ATM, right?

Reply:
Well, maybe, if you are afraid of breaking existing user configs. But they can still change the defaults without a PR in the future with an `advice-add`. My preference now is to not introduce too many deffaces until someone asks for them.

Reply:
I see no problems add one or more faces, my concern is we break someone else config because of that change, we could have a define-obsolete-variable for the old var just to keep things working for users that are already using that var

Reply:
> Well, we aren't customizing those either now right? When there's a need, someone somewhere will introduce a few new deffaces for the sizes.

We do, the variable is public and it is used for customizing the popup appearance. Similar variables exist for other packages  using posframe, see dap-mouse-posframe-properties, company-posframe-show-params .

Reply:
Fixed